src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/JsonContent.cs

```diff
@@ -20,8 +20,8 @@ public sealed partial class JsonContent : HttpContent
         private static MediaTypeHeaderValue DefaultMediaType
             => new MediaTypeHeaderValue(JsonMediaType) { CharSet = "utf-8" };
 
-        internal static JsonSerializerOptions DefaultSerializerOptions
-            => new JsonSerializerOptions { PropertyNameCaseInsensitive = true, PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
+        internal static readonly JsonSerializerOptions s_defaultSerializerOptions
```

Comment:
What prevents the caches inside of this instance from growing unbounded?  Do we need to place a limit on the size of the cache, making it an LRU cache or something like that?

Reply:
> What prevents the caches inside of this instance from growing unbounded? 

As of today, there is no bound  in the `JsonSerializerOptions` cache. 

> Do we need to place a limit on the size of the cache, making it an LRU cache or something like that?

Maybe, but the same problem occurs today on the cached options of

Reply:
Would it be better to address the bounded cache discussion in a separate issue since that has a larger scope?

Reply:
> Maybe, but the same problem occurs today on the static options used by the JsonSerializer directly.

Ok.  If this is matching what the serializer already does by default, then go ahead with it.  But it's probably worth having a separate discussion.